src/main/java/org/sciborgs1155/robot/Robot.java

```diff
@@ -58,18 +66,47 @@ private void configureGameBehavior() {
     }
   }
 
+  /** Creates an input stream for a joystick. */
+  private InputStream createJoystickStream(InputStream input, double maxSpeed, double maxRate) {
+    return input
+        .deadband(Constants.DEADBAND, 1)
```

Comment:
is the max needed here? Shouldn't in just never be more than 1 anyway?

Reply:
max is needed because the wpiutils deadband function re-scales the input from 0 to the max after deadbanding